docs/learn/dapp-developer/siwe.md

```diff
@@ -109,28 +236,7 @@ Resources:
 
 </details>
 
-## Verify the signature on the user's Universal Profile
-
-Your dApp has now received a message signed by the controller address of the Universal Profile. To finalise the login, you need to verify if the message was signed by an address which has the [`SIGN`](../../standards/universal-profile/lsp6-key-manager.md#permissions) permission for this UP.
-
-To do so, you can use the [`isValidSignature(...)`](../../contracts/contracts/UniversalProfile.md#isvalidsignature) function ([EIP-1271](https://eips.ethereum.org/EIPS/eip-1271)) to check the signature.
-
-```js
-// If the signature is valid it should return the success value 0x1626ba7e.
-// Then, the message was signed by an EOA which has a SIGN (https://github.com/lukso-network/standards/universal-profile/lsp6-key-manager#permissions)permission for this Universal Profile.
-// For additional details, check https://eips.ethereum.org/EIPS/eip-1271
-const isValidSignature = await myUniversalProfileContract.methods
-  .isValidSignature(hashedMessage, signature)
-  .call();
-
-if (isValidSignature === '0x1626ba7e') {
-  console.log('Log In successful!');
-} else {
-  // The EOA which signed the message has no SIGN permission over this UP.
-  console.log('Log In failed');
-}
-```
-
-### Additional Resources
+One practical application of the raw SIWI message is to generate _JSON Web Tokens_ after verifying the user's signature. This approach ensures that only the rightful owner of an Ethereum address can obtain a session token, enhancing the security and privacy of your Web3-based application. You can get additional guidance on the following resources:
 
 - [🎥 BuildUP #2 | Web3 Auth: Using Universal Profile and SIWE for Login (YouTube)](https://www.youtube.com/watch?v=lY18lBu3_XA)
+- [🛠️ LUKSO SIWE Demo Repository | Generating JWT's using Universal Profiles (GitHub)](https://github.com/richtera/lukso-siwe-demo)
```

Comment:
imo this repo is too complex and advanced for this - id rather put a link to the playground instead